sign-in.txt

```diff
@@ -8,9 +8,10 @@
   Brian Bansenauer  |  Bansenauer-Cascadia
   Roger L           |  Floydfix
   Gori Jesionowski  |  gjesionowski
-  Edmund Chan       |  edmundchan8
+  Edmund Chan       |   edmundchan8
```

Comment:
RIchard, it looks like you may have accidentally added a space in front of edmundchan8 in line 11. 
Please remove this so that the name lines up with the rest. Thanks.

Reply:
Darn I got caught